### DISCUSSION TITLE
wofi-like minimalist theme?

### DISCUSSION THREAD

**Discussion by mcepl** [davatorium/rofi#1711] [Theming]
I am running rofi on sway (with https://github.com/lbonn/rofi) and everything works fine (I was especially after https://github.com/plattfot/pinentry-rofi, because all other pinentry frontends suck in various ways). The only problem I have is that rofi looks weird on my system. wofi provides quite minimalistic look which I got used:

![screenshot-2022-09-21_14-09-1663763135](https://user-images.githubusercontent.com/198999/191503238-01254ea8-c185-4368-a7a3-7b0c98541e2f.png)

but when I run the similar program with rofi it looks weird no matter what theme I try (the default and `Arc.rasi` seem like the least offensive ones).

![screenshot-2022-09-21_14-09-1663763497](https://user-images.githubusercontent.com/198999/191504723-653a9232-3d57-42f7-a2cf-448b7c90b578.png)

Would anybody know about such minimalist wofi-like theme?

**Comment by DaveDavenport** [comment]
Given this is very subjective and you use unhelpful and undefined language like 'weird', 'offensive' .. no idea what theme fits your taste. 

That said, it should be trivial to recreate the first screenshot you posted in rofi.
